roles/openshift_version/tasks/first_master_containerized_version.yml

```diff
@@ -7,15 +7,15 @@
   when:
   - openshift_image_tag is defined
   - openshift_image_tag != ""
-  - openshift_version is not defined
+  - openshift_version is not defined or openshift_version == ""
```

Comment:
The 'not openshift_version_reinit' boolean below this one doesn't make any sense here.  I think we should combine that like this:
```yaml
- openshift_version is not defined or openshift_version_reinit | bool
```
We can define openshift_version_reinit in role defaults.